src/test/java/gov/healthit/chpl/aqa/pageObjects/SurveillanceReportPage.java

```diff
@@ -10,64 +10,128 @@
 public final class SurveillanceReportPage extends BasePage {
     private SurveillanceReportPage() {}
 
-    public static WebElement surveillancePageTitle(final WebDriver driver) {
-        return driver.findElement(By.xpath("//h1[contains(text(),'CHPL Surveillance')]"));
+    public static WebElement confirmMessage(final WebDriver driver) {
+        return driver.findElement(By.xpath("/html/body/div[1]/div/div/div[2]"));
     }
 
-    public static WebElement oncAcbs(final WebDriver driver, final String acbName) {
-        return driver.findElement(By.xpath("//div[@id='onc-acb-" + acbName + "']"));
+    public static WebElement deleteReportButton(final WebDriver driver) {
+        return driver.findElement(By.id("quarterly-suveillance-delete"));
+    }
+
+    public static WebElement editSurveillanceDataButton(final WebDriver driver, final String friendlySurvId) {
+        return driver.findElement(By.xpath("//*[@id=\"edit-surveillance-data-" + friendlySurvId + "\"]/i"));
+    }
+
+    public static WebElement editSurveillanceReport(final WebDriver driver, final String acbNameYearQuarter) {
+        return driver.findElement(By.xpath("//button[@id='act-" + acbNameYearQuarter + "']"));
+    }
+
+    public static WebElement groundsForInitiatingInput(final WebDriver driver) {
+        return driver.findElement(By.id("grounds-for-initiating"));
+    }
+
+    public static WebElement initiateButton(final WebDriver driver, final String acbNameYearQuarter) {
+        return driver.findElement(By.xpath("//button[@id='initiate-" + acbNameYearQuarter + "']"));
```

Comment:
If there's an ID, is the xpath required? Same with editSurveillanceReport and editSurveillanceDataButton

Reply:
Same, this one was also already there. I shall review all the page objects and update